tasks/pkg-debian.yml

```diff
@@ -2,21 +2,16 @@
 - name: Install apt-transport-https
   apt: name=apt-transport-https state=present
 
-- name: Install ubuntu apt-key server
-  apt_key: id=A2923DFF56EDA6E76E55E492D3A80E30382E94DE keyserver=hkp://keyserver.ubuntu.com:80 state=present
-  when: datadog_apt_key_url_new is not defined
-
-- name: Install Datadog apt-key
-  apt_key: id=A2923DFF56EDA6E76E55E492D3A80E30382E94DE url={{ datadog_apt_key_url_new }} state=present
-  when: datadog_apt_key_url_new is defined
-
-- name: Ensure ubuntu apt-key server is present
-  apt_key: id=C7A7DA52 keyserver=hkp://keyserver.ubuntu.com:80 state=present
-  when: datadog_apt_key_url is not defined
-
-- name: Ensure Datadog apt-key is present
-  apt_key: id=C7A7DA52 url={{ datadog_apt_key_url }} state=present
-  when: datadog_apt_key_url is defined
+- name: Ensure Datadog apt-keys are installed
+  apt_key:
+    id: "{{ item }}"
+    keyserver: "{{ datadog_apt_key_url }}"
```

Comment:
Good point, I overlooked that one set was using the `url` param and the other the`keyserver` param.

It's late, so maybe I'm missing something, but even when using `with_dict`, I'm not seeing how this could be DRY'd up and still retain the full logic of using a `keyserver` only for the keys where the custom `url` isn't specified? How do you handle the case where someone specifies a custom url for one of the keys but not the other? Or can we assume it's all-or-nothing that for both keys they'll either use the keyserver or specify custom urls?